build.py

```diff
@@ -169,6 +171,44 @@ def validate_actions(user_actions, valid_targets):
             sys.exit(1)
 
 
+def generate_mo(extraction_target_path, lang_name, trans_mappings, domain, localedir=None):
```

Comment:
We should separate out the po and mo generation, but we can do this in another PR.